client/src/index.jsx

```diff
@@ -1,6 +1,214 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import App from './components/App.jsx';
+import React from "react";
+import ReactDOM from "react-dom";
+import App from "./components/App.jsx";
 
+const languages = [
+	"cesky",
+	"Dansk",
+	"Deutsch",
+	"English",
+	"Español",
+	"Français",
+	"Italiano",
+	"Nederlands",
+	"Norsk",
+	"Polski",
+	"Portuês",
+	"Português (Brasil)",
+	"Svenska",
+	"Suomi",
+	"Türk"
+];
 
-ReactDOM.render(<App />, document.getElementById('app'));
\ No newline at end of file
+const currency = [
+	"EUR",
+	"GBP",
+	"USD",
+	"---",
+	"AED",
+	"ALL",
+	"AMD",
+	"ARS",
+	"AUD",
+	"AZN",
+	"BDT",
+	"BHD",
+	"BND",
+	"BOB",
+	"BRL",
+	"BSD",
+	"BZD",
+	"CAD",
+	"CHF",
+	"CLP",
+	"CNY",
+	"COP",
+	"CRC",
+	"CUC",
+	"CZK",
+	"DJF",
+	"DKK",
+	"DOP",
+	"DZD",
+	"EGP",
+	"EUR",
+	"FJD",
+	"GBP",
+	"HKD",
+	"HNL",
+	"HRK",
+	"HUF",
+	"IDR",
+	"ILS",
+	"INR",
+	"ISK",
+	"JOD",
+	"JPY",
+	"KES",
+	"KGS",
+	"KES",
+	"KGS",
+	"KHR",
+	"KMF",
+	"KRW",
+	"KWD",
+	"KZT",
+	"LAK",
+	"LBP",
+	"LKR",
+	"MAD",
+	"MNT",
+	"MRO",
+	"MUR",
+	"MXN",
+	"MYR",
+	"NAD",
+	"NIO",
+	"NOK",
+	"NPR",
+	"NZD",
+	"OMR",
+	"PEN",
+	"PGK",
+	"PHP",
+	"PKR",
+	"PLN",
+	"PYG",
+	"QAR",
+	"RON",
+	"RUB",
+	"RWF",
+	"SAR",
+	"SEK",
+	"SGD",
+	"SOS",
+	"SZL",
+	"THB",
+	"TND",
+	"TOP",
+	"TRY",
+	"TTD",
+	"TWD",
+	"TZS",
+	"UAH",
+	"UGX",
+	"USD",
+	"UYU",
+	"VEF",
+	"VND",
+	"VUV",
+	"WST",
+	"XAF",
+	"XCD",
+	"XOF",
+	"XPF",
+	"ZAR"
+];
+
+const guests = [
+	"1 Guest",
```

Comment:
@enague This can be shortened a little more by using the for loop from 1 to 100 for example, then render the options accordingly

Reply:
ok will consider refactoring later! thanks for the suggestion!